paper-autocomplete.html

```diff
@@ -206,6 +206,43 @@
 </dom-module>
 <script>
   (function () {
+    //---------------------------------------------------------------------------
+    //                  IE11 function polyfills to work properly
+    //---------------------------------------------------------------------------
+
+    if (window.NodeList && !NodeList.prototype.forEach) {
+      NodeList.prototype.forEach = function (callback, argument) {
+          argument = argument || window;
+          for (var i = 0; i < this.length; i++) {
+              callback.call(argument, this[i], i, this);
+          }
+      };
+    }
+
+    if (!Array.prototype.find) {
```

Comment:
In fact, I had already replace your .find() method with filter(), so this is not needed